WalletWasabi/Blockchain/Keys/HdPubKeyPathView.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace WalletWasabi.Blockchain.Keys;
+
+public class HdPubKeyPathView : IEnumerable<HdPubKey>
+{
+	internal HdPubKeyPathView(IEnumerable<HdPubKey> hdPubKeys)
```

Comment:
No in this case because a `HdPubKeyPathView` should be created for a  `HdPubKeyGlobalView` and the global view internal state is immutable so, transitively the `IEnumerable<>` here is an enumeration over an immutable data structure, what is okay and i don't have to invoke `ToImmutableList()` again and again.

Reply:
Yes, it can be. I will make it immutable then, after all it is always preferable.